en/docs/get-started/quickstart.md

```diff
@@ -31,7 +27,7 @@ See the instructions on [managing users in organizations](../../guides/org-user-
 
 ## Step 4: Share organization details
 
-Once the organization admins are defined, you (as the super administrator) need to share the access URL of the sub organizations with the relevant administrators.
+Once the organization admins are defined, you (as the super administrator) need to share the access URL of the sub organizations with the relevant administrators in each sub organization.
```

Comment:
**admins** and **administrators** are used interchangeably. Should we stick to a single term?

Reply:
+1. Will send another PR for that.